### PR TITLE
chore: maintain proto setup

### PR DESCRIPTION
**Pull request by faddat**
This PR:

* uses the latest proto-builder
* pins the versions that we use from the `buf` registry 
* rebuilds the .proto files

### CODE REVIEW

File: x/wasm/types/authz.pb.go [hunk 1]
```diff
@@ -5,25 +5,22 @@ package types
 
 import (
 	fmt "fmt"
-	io "io"
```
Comment (alpe) [inline x/wasm/types/authz.pb.go:8]: Can you run `make format` to have the imports formatted well?
fyi: `make proto-all` should be the preferred target for generation. It includes linter and formatting

**Review by alpe — CHANGES_REQUESTED** [review]
Nice work. Thank you for the help!
Just formatting, then it is good to be merged

File: x/wasm/types/authz.pb.go (hunk 1 above)
Comment (faddat) [reply x/wasm/types/authz.pb.go:8]: :) sounds great

**Review by pinosu — APPROVED** [review]
Thanks for your contribution! LGTM 👍

**Review by alpe — APPROVED** [review]
nice work! 🏅

**Pull request merged**